TestingLowerBounds/KullbackLeibler.lean

```diff
@@ -91,8 +91,18 @@ lemma kl_eq_fDiv [SigmaFinite μ] [SigmaFinite ν] :
 lemma kl_self (μ : Measure α) [SigmaFinite μ] : kl μ μ = 0 := by
   rw [kl_eq_fDiv, fDiv_self (by norm_num)]
 
+lemma kl_eq_top_iff [IsFiniteMeasure μ] [SigmaFinite ν] :
+    kl μ ν = ⊤ ↔ μ ≪ ν → ¬ Integrable (llr μ ν) μ := by
+  rw [kl_eq_fDiv, fDiv_mul_log_eq_top_iff]
+
 section kl_nonneg
 
+lemma kl_ne_bot (μ ν : Measure α) : kl μ ν ≠ ⊥ := by
```

Comment:
Is the proof shorter if you use `fDiv_ne_bot_of_derivAtTop_nonneg`?

Reply:
Yes, if I use that lemma then I can finish the proof in a single line using 

simp only [kl_eq_fDiv, ne_eq, derivAtTop_mul_log, le_top, fDiv_ne_bot_of_derivAtTop_nonneg,
    not_false_eq_true]

But in this case I would have to add the hypothesis that the measures are sigma finite, while the lemma is true in general without them.

Should I keep the current version or modify it to make it shorter?

Reply:
Also, should it have the simp attribute?

Reply:
Less assumptions is better. `simp` is a good idea.